### PR TITLE
cis bench stuff

### PR DESCRIPTION
**Pull request by SoundGoof**

### CODE REVIEW

**Review by sonix-tisteagle — COMMENTED** [review]
Looks good, however, we need to discuss the available hard drive space..

File: rocky9.ks.cfg [hunk 1]
```diff
@@ -43,18 +52,142 @@ clearpart --all --initlabel
...
+echo "MaxAuthTries 4" >> /etc/ssh/sshd_config.d/sonix.conf
+echo "MaxStartups 10:30:60" >> /etc/ssh/sshd_config.d/sonix.conf
+echo "MaxSessions 10" >> /etc/ssh/sshd_config.d/sonix.conf
+echo "LoginGraceTime 60" >> /etc/ssh/sshd_config.d/sonix.conf
+echo "ClientAliveInterval 1200" >> /etc/ssh/sshd_config.d/sonix.conf
+echo "ClientAliveCountMax 0" >> /etc/ssh/sshd_config.d/sonix.conf
+echo "Defaults use_pty" > /etc/sudoers.d/sonix-pty
+echo 'Defaults logfile="/var/log/sudo.log"' > /etc/sudoers.d/sonix-logfile
+
+echo "minclass = 4" > /etc/security/pwquality.conf.d/sonix-polixy.conf
```
Comment (bluecmd) [inline rocky9.ks.cfg]: Typo on the file name?

File: rocky9.ks.cfg [hunk 2]
```diff
@@ -43,18 +52,142 @@ clearpart --all --initlabel
...
+EOF
+
+echo "/dev/shm                      tmpfs         defaults,rw,nosuid,nodev,noexec,relatime   0 0" >> /etc/fstab
+echo "install usb-storage /bin/true" > /etc/modprobe.d/usbstorage.conf
+chmod og-rwx /boot/grub2/grub.cfg
+echo "Storage=none" >> /etc/systemd/coredump.conf
+echo "ProcessSizeMax=0" >> /etc/systemd/coredump.conf
+
+echo "net.ipv6.conf.all.autoconf = 0" > /etc/sysctl.d/sonix-ipv6.conf
+echo "net.ipv6.conf.all.disable_ipv6 = 1" >> /etc/sysctl.d/sonix-ipv6.conf
```
Comment (bluecmd) [inline rocky9.ks.cfg]: This one we probably want enabled

File: rocky9.ks.cfg (hunk 1 above)
Comment (SoundGoof) [reply rocky9.ks.cfg]: fixxed

File: rocky9.ks.cfg (hunk 2 above)
Comment (SoundGoof) [reply rocky9.ks.cfg]: removed disabeling of v6

**Review by bluecmd — APPROVED** [review]

**Pull request merged**